### PR TITLE
Update README to include the webapp

### PR DESCRIPTION
**Pull request by thealphadollar**
A webapp was developed by Nishant to aid the process of getting time table from ERP.

Not everyone would want to clone the repository and run the application on their system and hence it is better to include the link to the currently running web app.

I'll try to add some more features to the webapp and see if it's possible to host it on my DO.

### CODE REVIEW

**Comment by thealphadollar** [comment]
@ghostwriternr @nishnik Plese review.

File: README.md [hunk 1]
```diff
@@ -15,7 +15,25 @@ you an **ICS file** which you can add in any common calendar application.
 
 - **Step 1:** Get your timetable from ERP:
 
+### Web Application
+
+  Go to
+
+  ```html
+
+  http://gyftkgp.herokuapp.com/
```
Comment (nishnik) [inline README.md]: It won't work without https. Change this!

**Comment by thealphadollar** [comment]
@nishnik Done.

File: README.md [hunk 2]
```diff
@@ -15,7 +15,25 @@ you an **ICS file** which you can add in any common calendar application.
...
+  Go to
+
+  ```html
+
+  https://gyftkgp.herokuapp.com/
+  ```
+
+  Enter your roll number and get the security question.
+
+  Once the security question is fetch, enter the credentials correctly and save the ICS file.
```
Comment (icyflame) [inline README.md]: grammar

File: README.md [hunk 3]
```diff
@@ -15,7 +15,25 @@ you an **ICS file** which you can add in any common calendar application.
 
 - **Step 1:** Get your timetable from ERP:
 
+### Web Application
+
+  Go to
+
+  ```html
+
+  https://gyftkgp.herokuapp.com/
```
Comment (icyflame) [inline README.md]: use markdown links instead of preformatted text

**Comment by icyflame** [comment]
@nishnik please also add the link to the repository that contains the code for the webapp. (I would prefer it if you would move the repository to the metakgp org, instead of keeping it as a personal repo. I have moved some of my mature projects to the metakgp domain and that automatically gives some permissions to other people. You may also decide to wait for a while and then move, after it's a bit more stable, say a month or 2 while you work on it. :slightly_smiling_face: )

Thanks for the patch, @thealphadollar!

**Comment by thealphadollar** [comment]
@icyflame @nishnik updated.

**Comment by nishnik** [comment]
Of course, I don't see myself working much (except for few little tweaks) on `gyft-serve`. I was going to handle it over during vacations, would like to improve the UX a bit more.

**Pull request merged**